lib/gmail/message_attachment.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Gmail.MessageAttachment do
+
+  @moduledoc """
+  An email message attachment.
+  """
+
+  alias __MODULE__
+  alias Gmail.{Helper}
+  import Gmail.Base
+
+  @doc """
+  Gmail API documentation: https://developers.google.com/gmail/api/v1/reference/users/messages/attachments
+  """
+  defstruct attachmentId: "",
+    size: "",
+    data: ""
+
+  @type t :: %__MODULE__{}
+
+  @doc """
+  Gets the specified attachment.
+
+  Gmail API documentation: https://developers.google.com/gmail/api/v1/reference/users/messages/attachments/get
+  """
+  @spec get(String.t, String.t, String.t) :: {atom, String.t, String.t}
+  def get(user_id, message_id, id) do
+    path = querify_params("users/#{user_id}/messages/#{message_id}/attachments/#{id}", [], %{})
+
+    {:get, base_url, path}
+  end
+
+  @doc """
+  Converts a Gmail API attachment resource into a local struct.
+  """
+  @spec convert(map) :: MessageAttachment.t
```

Comment:
@talon yeah, I think it's `dialyzer`. Looking into it now 😹 - learning in public